packages/apps/faucet-server/src/constants/networks.ts

```diff
@@ -29,10 +29,6 @@ export const FAUCET_NETWORKS: {
   [ChainId.XLAYER_TESTNET]: {
     rpcUrl: process.env.RPC_URL_XLAYER_TESTNET || '',
   },
-  [ChainId.SKALE]: {
```

Comment:
since faucetAddress is not used anymore, the variable can be removed from IFaucetData

Reply:
removed